packages/rich-text-editor/test/basic.test.js

```diff
@@ -928,4 +928,34 @@ describe('rich text editor', () => {
       );
     });
   });
+
+  describe('listener clean up', () => {
+    it('should not have active listeners once detached', () => {
+      expect(editor.emitter).to.not.equal(null);
+      expect(editor.emitter._events).to.not.be.empty;
+      expect(editor.emitter._eventsCount).to.greaterThan(0);
+      expect(editor.emitter.listeners).to.not.be.empty;
+
+      rte.parentNode.removeChild(rte);
+
+      expect(editor.emitter._events).to.be.empty;
+      expect(editor.emitter._eventsCount).to.be.equal(0);
+      expect(editor.emitter.listeners).to.be.empty;
+    });
+
+    it('should have the listeners when removed and added back again', () => {
+      const parent = rte.parentNode;
+
+      parent.removeChild(rte);
+      parent.appendChild(rte);
+
+      const __rte = document.querySelector('vaadin-rich-text-editor');
+      const __editor = __rte._editor;
```

Comment:
There is no need to make another `querySelector` as the existing `rte` reference can be used.